packages/edgebridge/README.md

```diff
@@ -0,0 +1,142 @@
+
+For sending the data to Analytics Report, Add Service `Adobe Analytics` to Datastreams when configurating. 
+
+> **Note**  
+> Experience Platform Edge Bridge does not have a corresponding extension card in the Data Collection UI; no changes to a Data Collection mobile property are required to use Edge Bridge.
+
+Then registering the Edge Bridge extension with the Mobile Core.
+Note that initializing the SDK should be done in native code, additional documentation on how to initialize the SDK can be found [here](https://github.com/adobe/aepsdk-react-native#initializing).
+
+
+**Initialization Example**
+
+iOS
+```objc
+// AppDelegate.h
+@import AEPCore;
+@import AEPEdge;
+@import AEPEdgeIdentity;
+@import AEPEdgeBridge;
+...
+@implementation AppDelegate
+
+// AppDelegate.m
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+
+   // TODO: Set up the preferred Environment File ID from your mobile property configured in Data Collection UI
+   NSString* ENVIRONMENT_FILE_ID = @"YOUR-APP-ID";
+
+   NSArray *extensionsToRegister = @[AEPMobileEdgeIdentity.class, 
+                                     AEPMobileEdge.class,
+                                     AEPMobileEdgeBridge.class
+                                     ];
+
+   [AEPMobileCore configureWithAppId: ENVIRONMENT_FILE_ID];  
+    ...   
+  }]; 
+  return YES;   
+ } 
+
+@end
+```
+
+Android
+```java
+import com.adobe.marketing.mobile.MobileCore;
+import com.adobe.marketing.mobile.Edge;
+import com.adobe.marketing.mobile.edge.identity.Identity;
+import com.adobe.marketing.mobile.edge.bridge.EdgeBridge
+
+...
+import android.app.Application;
+...
+public class MainApplication extends Application implements ReactApplication {
+  ...
+  // TODO: Set up the preferred Environment File ID from your mobile property configured in Data Collection UI
+  private final String ENVIRONMENT_FILE_ID = "YOUR-APP-ID";
+
+  @Override
+  public void on Create(){
+    super.onCreate();
+    ...
+  
+    MobileCore.setApplication(this);
+    MobileCore.configureWithAppID(ENVIRONMENT_FILE_ID);
+
+    MobileCore.registerExtensions(
+      Arrays.asList(Identity.EXTENSION, Edge.EXTENSION, EdgeBridge.EXTENSION),
+      o -> Log.d("MainApp", "Adobe Experience Platform Mobile SDK was initialized")
+    );
+  }
+}  
+```
+
+### Importing the extension
+In your React Native application, import the Edge extension as follows:
+```typescript
+import {EdgeBridge} from '@adobe/react-native-aepedgebridge';
+```
+
+### Validating and Data Prep Mapping
+
+Refer to [Validating Event in Assurance tutorial](https://github.com/adobe/aepsdk-edgebridge-ios/blob/main/Documentation/tutorials/edge-bridge-tutorial.md#initial-validation-with-assurance) for Edge Bridge events examples.
+
+Refer to [Data Prep Mapping tutorial](https://github.com/adobe/aepsdk-edgebridge-ios/blob/main/Documentation/tutorials/edge-bridge-tutorial.md#data-prep-mapping) for Data Prep Mapping examples.
```

Comment:
Let's combine these into a single link to the existing tutorial, so in case these headers/anchors change in the future it does not affect this doc. E.g.:

For Data Prep mapping examples and tips on validating with Assurance, refer to the [Edge Bridge tutorials](https://github.com/adobe/aepsdk-edgebridge-ios/tree/main/Documentation/tutorials).